operations/validate_csm_health.md

```diff
@@ -572,22 +569,34 @@ Overall Gateway Test Status:  PASS
 
 For more detailed information on the tests results and examples, see [Gateway Testing](network/gateway_testing.md).
 
-<a name="booting-csm-barebones-image"></a>
+The gateway tests can be run from various locations. For this part of the CSM validation, check gateway access from the NCNs and from outside the system.
+Externally, the API gateway is accessible on the CMN and either the CAN or CHN, depending on the configuration of the system.
+On NCNs, the API gateway is accessible on the same networks (CMN and CAN/CHN) and it is also accessible on the NMNLB network.
+
+#### 4.1.2 Gateway health tests on an NCN
+
+The gateway tests may be run on any NCN with the `docs-csm` RPM installed. For details on installing the `docs-csm` RPM, see [Check for Latest Documentation](../update_product_stream/index.md#check-for-latest-documentation).
+
+To execute the tests, see [Running Gateway Tests on an NCN Management Node](network/gateway_testing.md#running-gateway-tests-on-an-ncn-management-node).
+
+#### 4.1.3 Gateway health tests from outside the system
+
+To execute the tests, see [Running Gateway Tests on a Device Outside the System](network/gateway_testing.md#running-gateway-tests-on-a-device-outside-the-system).
 
 ### 4.2 Internal SSH access test execution
 
 The internal SSH access tests may be run on any NCN with the `docs-csm` RPM installed. For details on installing the `docs-csm` RPM,
-see [Check for Latest Documentation](../update_product_stream/index.md#documentation).
+see [Check for Latest Documentation](../update_product_stream/index.md#check-for-latest-documentation).
 
 Execute the tests by running the following command:
 
 ```bash
 ncn# /usr/share/doc/csm/scripts/operations/pyscripts/start.py test_bican_internal
 ```
 
-By default, SSH access will be tested between master nodes and spine switches on all relevant networks.
-It is possible to customize which nodes and networks will be tested. For example, you may wish to include configured
-UANs as part of the tests. See the test usage statement for details. The test usage statement is displayed by calling the
+By default, SSH access will be tested on all relevant networks between master nodes, spine switches, compute nodes, and UANs.
```

Comment:
By default, the SSH access is only tested between master nodes and spine switches. This change was made because I was made aware that UANs/computes are most likely not set up at this point.